code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -137,5 +137,9 @@
 		var/mob/living/L = source
 		if(fake_species && L.health > disguise_fail_health)
 			switch(fake_species.type)
+			//SKYRAT EDIT REMOVAL BEGIN
```

Comment:
Tbh, drop the invasive non-modular changes like this.

Reply:
now that i think about it, i can just make the bananium golem uncraftable